### PR TITLE
Release notes v5.1.0-preview2

### PR DESCRIPTION
**Pull request by lcheunglci**
Add release notes for 5.1.0preview2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [Preview Release 5.1.0-preview2.XXX.X] - 2022-11-09
+
+This update brings the below changes over the previous release:
+
+### Breaking changes over preview release v5.1.0-preview2
```
Comment (ErikEJ) [inline CHANGELOG.md:11]: preview1 ??

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+
+- Add support for .NET 6.0 and Dropped support for .NET Core 3.1 [#1704](https://github.com/dotnet/SqlClient/pull/1704)
+
+### Fixed
+
+- Fixed memory leak regression from [#1781](https://github.com/dotnet/SqlClient/pull/1781) using a `DisposableTemporaryOnStack` struct. [#1818](https://github.com/dotnet/SqlClient/pull/1818)
+
+### Changed
+
+- Updated `Microsoft.Data.SqlClient.SNI` (.NET Framework dependency) and `Microsoft.Data.SqlClient.SNI.runtime` (.NET Core/Standard dependency) version to `5.1.0-preview2.22311.2`. [#1831](https://github.com/dotnet/SqlClient/pull/1831) which includes TLS 1.3 fix timeout issue and double handshake, remove ARM32 binaries, and support for `ServerCertificate` [#1822](https://github.com/dotnet/SqlClient/issues/1822)
```
Comment (ErikEJ) [inline CHANGELOG.md:21]: Fix for timeout issue

File: release-notes/5.1/5.1.0-preview2.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.24.0
+- System.Buffers 4.5.1
+- System.Configuration.ConfigurationManager 6.0.1
+- System.IO 4.3.0
+- System.Runtime.InteropServices.RuntimeInformation 4.3.0
+- System.Security.Cryptography.Algorithms 4.3.1
+- System.Security.Cryptography.Primitives 4.3.0
+- System.Text.Encoding.Web 6.0.0
+
+#### .NET Core
```
Comment (David-Engel) [inline release-notes/5.1/5.1.0-preview2.md:52]:
```suggestion
#### .NET
```

**Review by David-Engel — APPROVED** [review]

File: release-notes/5.1/5.1.0-preview2.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+### Breaking changes over preview release v5.1.0-preview1
+
+- Add support for .NET 6.0 and Dropped support for .NET Core 3.1 [#1704](https://github.com/dotnet/SqlClient/pull/1704)
+
+### Added
+
+- Added support for `DateOnly` and `TimeOnly` for `SqlParameter` value and `GetFieldValue`. [#1813](https://github.com/dotnet/SqlClient/pull/1813)
+- Added support for TLS 1.3 for .NET Core and SNI Native. [#1821](https://github.com/dotnet/SqlClient/pull/1821)
+- Added `ServerCertificate` support for `Encrypt=Mandatory` or `Encrypt=Strict`. [#1822](https://github.com/dotnet/SqlClient/pull/1822)
```
Comment (DavoudEshtehari) [inline release-notes/5.1/5.1.0-preview2.md:15]: Can you add a new section and include a sample for the new parameter?

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,31 @@ All notable changes to this project will be documented in this file.
...
+- Added Windows ARM64 support when targeting .NET Framework. [#1828](https://github.com/dotnet/SqlClient/pull/1828)
+
+### Fixed
+
+- Fixed memory leak regression from [#1781](https://github.com/dotnet/SqlClient/pull/1781) using a `DisposableTemporaryOnStack` struct. [#1818](https://github.com/dotnet/SqlClient/pull/1818)
+
+### Changed
+
+- Updated `Microsoft.Data.SqlClient.SNI` (.NET Framework dependency) and `Microsoft.Data.SqlClient.SNI.runtime` (.NET Core/Standard dependency) version to `5.1.0-preview2.22311.2`. [#1831](https://github.com/dotnet/SqlClient/pull/1831) which includes the fix for the TLS 1.3 timeout and double handshake issue, removal of ARM32 binaries, and support for the `ServerCertificate` option. [#1822](https://github.com/dotnet/SqlClient/issues/1822)
+- Reverted "Excluding unsupported TLS protocols" for issue [#1151](https://github.com/dotnet/SqlClient/issues/1151) (i.e. removed `Switch.Microsoft.Data.SqlClient.EnableSecureProtocolsByOS`). [#1824](https://github.com/dotnet/SqlClient/issues/1824)
```
Comment (DavoudEshtehari) [inline CHANGELOG.md:29]:
```suggestion
- Reverted "Excluding unsupported TLS protocols" for issue [#1151](https://github.com/dotnet/SqlClient/issues/1151) (i.e. removed `Switch.Microsoft.Data.SqlClient.EnableSecureProtocolsByOS`) by adding support for TLS 1.3. [#1824](https://github.com/dotnet/SqlClient/issues/1824)
```

File: release-notes/5.1/5.1.0-preview2.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+- Fixed memory leak regression from [#1781](https://github.com/dotnet/SqlClient/pull/1781) using a `DisposableTemporaryOnStack` struct. [#1818](https://github.com/dotnet/SqlClient/pull/1818)
+
+### Changed
+
+- Updated `Microsoft.Data.SqlClient.SNI` (.NET Framework dependency) and `Microsoft.Data.SqlClient.SNI.runtime` (.NET Core/Standard dependency) version to `5.1.0-preview2.22311.2`. [#1831](https://github.com/dotnet/SqlClient/pull/1831) which includes the fix for the TLS 1.3 timeout and double handshake issue, removal of ARM32 binaries, and support for the `ServerCertificate` option. [#1822](https://github.com/dotnet/SqlClient/issues/1822) [Read more](#server-certificate-support)
+- Reverted "Excluding unsupported TLS protocols" for issue [#1151](https://github.com/dotnet/SqlClient/issues/1151) (i.e. removed `Switch.Microsoft.Data.SqlClient.EnableSecureProtocolsByOS`) by adding support for TLS 1.3. [#1824](https://github.com/dotnet/SqlClient/issues/1824)
+- Code health improvements [#1812](https://github.com/dotnet/SqlClient/pull/1812) [#1520](https://github.com/dotnet/SqlClient/pull/1520) 
+
+## New features over stable release v5.0
```
Comment (DavoudEshtehari) [inline release-notes/5.1/5.1.0-preview2.md:28]: For consecutive preview versions, you can skip the stable version part. These notes will be cumulatived when a GA is released over them.
```suggestion
## New features
```

**Review by DavoudEshtehari — APPROVED** [review]

**Review by David-Engel — APPROVED** [review]

**Pull request merged**